### ISSUE TITLE
Gefühlte Inkonsistenz beim einschalten und aufheben der Interessant-Markierung

### ISSUE THREAD

**Issue by auge8472** [ckruse/cforum#788]
Habe ich einen Beitrag über den Button im Fuß des Beitrags als interessant markiert, wird die Box des Beitrags *bei seinem nächsten Aufruf* mit einem (für mich) irgendwie orangenen Farbton hinterlegt. Entferne ich nun die Interessant-Markierung, wird als Hintergrundfarbe *beim nächsten Aufruf* wieder das normale grau angezeigt.

Meine Überlegung ist nun folgende. Der Stern in der Anzeige des Threadbaums vor dem Betreff eines Beitrags wird on-the-fly beim markieren bzw. beim entfernen der Markierung gesetzt oder entfernt. Nicht nur aber besonders in der Nested-Ansicht des Threads ist der Threadbaum oft nicht im Sichtbereich, so dass eine Kontrolle, ob die Umschaltung auch erfolgt ist, das Scrollen zum Threadbaum erfordert. Ob bei Bedarf eine Fehlermeldung angezeigt wird, vermag ich allerdings nicht zu sagen, denn ich kann mich nicht erinnern, ob mir das setzen der Markierung schon einmal misslang.

Jedenfalls finde ich es logisch, die Hintergrundfarbe der Beitragsbox, wie den Stern im Threadbaum auch, bei Betätigung des Buttons on-the-fly umzuschalten.

**Comment by MatthiasApsel** [comment]
Du kannst es sofort am button-text erkennen. "als interessant markieren" ⇔ "Interessantmarkierung entfernen"

**Comment by auge8472** [comment]
> Du kannst es sofort am button-text erkennen. "als interessant markieren" ⇔ "Interessantmarkierung entfernen"

Ja, kann ich, weiß ich. Mir geht es hier primär um die "gefühlte Inkonsistenz", dass der Button und auch der Stern im Threadbaum sofort umschalten, die Hintergrundfarbe der Beitragsbox, das großflächigste und damit offensichtlichste Anzeichen einer erfolgten Umschaltung, aber nicht. Vermutlich – ich habe den betreffenden Quelltext nicht studiert – ist für die Hintergrundfarbe nur eine Klasse zu setzen oder zu entfernen, so, wie ich es für den Stern und die Button-Beschriftung vermute. Konistenter- und konsequenterweise sollte mMn auch die Hintergrundfarbe sofort umgeschaltet werden. Soweit meine Anregung.
